app/src/main/java/org/oppia/app/utility/RatioExtensions.kt

```diff
@@ -0,0 +1,15 @@
+package org.oppia.app.utility
+
+import android.content.Context
+import org.oppia.app.R
+import org.oppia.app.model.RatioExpression
+
+/**
+ * Returns this Ratio in string format.
+ * E.g. [1, 2, 3] will yield to 1 to 2 to 3
```

Comment:
Suggest:

``Returns an accessibly readable string representation of this [RatioExpression].``

Reply:
done